activesupport/lib/active_support/message_encryptor.rb

```diff
@@ -19,7 +19,17 @@ module ActiveSupport
   #   encrypted_data = crypt.encrypt_and_sign('my secret data')                  # => "NlFBTTMwOUV5UlA1QlNEN2xkY2d6eThYWWh..."
   #   crypt.decrypt_and_verify(encrypted_data)                                   # => "my secret data"
   class MessageEncryptor
-    DEFAULT_CIPHER = "aes-256-cbc"
+    class << self
+      attr_accessor :use_authenticated_message_encryption
+
+      def default_cipher
```

Comment:
Needs a `# :nodoc` same for the `use_authenticated_message_encryption` accessor.